### DISCUSSION TITLE
API | Leads+Documents

### DISCUSSION THREAD

**Discussion by Marshal4582** [YetiForceCompany/YetiForceCRM#16528] [How to] [answered]
Stuck on how to deal with creating Lead and the same time uploading attachment to Documents module. 
Can you gents guide me?
Using webservicesstandard

**Comment by rskrzypczak** [comment, marked as answer]
You cannot create two records with one query.
Check:
- https://yetiforce.com/en/knowledge-base/documentation/developer-documentation/item/how-to-add-a-document-with-an-attachment-via-rest-api
- https://github.com/YetiForceCompany/YetiForceCRM/discussions/15239

**Reply by Marshal4582** [reply]
Thanks
